packages/protocol/releaseData/initializationData/release3.json

```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
-  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0.0005", "0.01", "300", "1"]
-}
\ No newline at end of file
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "4000000000000000000000000"], "ExchangeEUR"],
```

Comment:
2M cEUR for the community fund at [0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972](https://explorer.celo.org/address/0xd533ca259b330c7a88f74e000a3faea2d63b7972/transactions)
5M cEUR to the foundation at [0x989e1a3B344A43911e02cCC609D469fbc15AB1F1](https://explorer.celo.org/address/0x989e1a3b344a43911e02ccc609d469fbc15ab1f1/transactions)

Reply:
You have 4M in the code. And I think we said 6M in the governance call